openapi/mx_platform_api_beta.yml

```diff
@@ -2113,14 +2113,14 @@ paths:
               schema:
                 "$ref": "#/components/schemas/ConnectWidgetResponseBody"
           description: OK
-      summary: Request connect widget URL
+      summary: Request connect widget url
       tags:
       - mx_platform
   "/users/{user_guid}/holdings":
     get:
       description: This endpoint returns all holdings associated with the specified
         `user` across all accounts and members.
-      operationId: listHoldingsByUser
+      operationId: listHoldings
```

Comment:
Curious why you decided to go with this particular naming? It looks like in other instances where we are listing by a `user`, `member`, etc. you are doing `listAccountOwnersByMember`. Should we leave this as `listHoldingsByUser` to follow the same pattern?

Reply:
We had a good chat about naming, but I will leave a few comments to explain our conclusion. Anything that is has a `{user_guid}` in the path is not going to include `ByUser`. The `By"resource"` will be added when there is an additional guid present in the path.